integration/e2emimir/client.go

```diff
@@ -1479,6 +1480,41 @@ func (c *Client) GetReceiversExperimental(ctx context.Context) ([]alertingmodels
 	return decoded, nil
 }
 
+func (c *Client) TestTemplatesExperimental(ctx context.Context, ttConf alertingNotify.TestTemplatesConfigBodyParams) (*alertingNotify.TestTemplatesResults, error) {
+	u := c.alertmanagerClient.URL("api/v1/grafana/templates/test", nil)
+
+	data, err := json.Marshal(ttConf)
+	if err != nil {
+		return nil, fmt.Errorf("error marshalling test templates config: %s", err)
+	}
+
+	req, err := http.NewRequest(http.MethodPost, u.String(), bytes.NewReader(data))
+	if err != nil {
+		return nil, fmt.Errorf("error creating request: %v", err)
```

Comment:
[Nit] Might as well be consistent :)
```suggestion
		return nil, fmt.Errorf("error creating request: %s", err)
```